apps/shell/Kconfig

```diff
@@ -49,5 +49,20 @@ config TASH_SCRIPT
 	---help---
 		This suppors parsing and executing shell script.
 		See "sh" on TASH help command.
+
+config SECURED_TASH
+    bool "Enable Secure shell"
+    depends on NET_SECURITY_TLS
+    default n
+
+if SECURED_TASH
+
+config TASH_PASSWORD_SHA256
+    string "SHA256 encrypted password"
+    default "samsung"
+    ---help---
+        Enable password to enter tash shell
```

Comment:
TASH is TinyAra SHell so that TASH shell is TinyAra SHell shell. Let's remove the word "shell".